example_apps/interal_sdk_inference_orchestrator_example_app/environment.yml

```diff
@@ -0,0 +1,17 @@
+name: example-inference-orchestrator
+channels:
+  - defaults
+dependencies:
+    - python=3.9.13
+    - pip=23.3
+    - pip:
+      - black
+      - coverage
+      - flake8
+      - isort
+      - pytest
+      - pre-commit
+      - sphinx
+      - python-magic
+      - pylint
+      - -r requirements.txt
```

Comment:
_requirement.txt_ not _requirements.txt_
failed when creating environment

Reply:
I just update the file name, please try again. Thank you